[2024-06-07T14:01:56.726+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-07T14:01:56.811+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: s3_postgres.create_postgres_table scheduled__2024-06-05T00:00:00+00:00 [queued]>
[2024-06-07T14:01:56.846+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: s3_postgres.create_postgres_table scheduled__2024-06-05T00:00:00+00:00 [queued]>
[2024-06-07T14:01:56.850+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-07T14:01:56.911+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): create_postgres_table> on 2024-06-05 00:00:00+00:00
[2024-06-07T14:01:56.928+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=50) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-07T14:01:56.937+0000] {standard_task_runner.py:63} INFO - Started process 63 to run task
[2024-06-07T14:01:56.952+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 's3_postgres', 'create_postgres_table', 'scheduled__2024-06-05T00:00:00+00:00', '--job-id', '824', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmptgeu4fq3']
[2024-06-07T14:01:56.958+0000] {standard_task_runner.py:91} INFO - Job 824: Subtask create_postgres_table
[2024-06-07T14:01:57.410+0000] {task_command.py:426} INFO - Running <TaskInstance: s3_postgres.create_postgres_table scheduled__2024-06-05T00:00:00+00:00 [running]> on host 1dcaa0c0151e
[2024-06-07T14:01:58.029+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='s3_postgres' AIRFLOW_CTX_TASK_ID='create_postgres_table' AIRFLOW_CTX_EXECUTION_DATE='2024-06-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-05T00:00:00+00:00'
[2024-06-07T14:01:58.040+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-07T14:01:58.128+0000] {base.py:84} INFO - Using connection ID 'postgres_default' for task execution.
[2024-06-07T14:01:58.663+0000] {sql.py:470} INFO - Running statement: 
    CREATE TABLE IF NOT EXISTS potential_customers (
        UserID TEXT,
        basket_icon_click INTEGER,
        basket_add_list INTEGER,
        basket_add_detail INTEGER,
        sort_by INTEGER,
        image_picker INTEGER,
        account_page_click INTEGER,
        promo_banner_click INTEGER,
        detail_wishlist_add INTEGER,
        list_size_dropdown INTEGER,
        closed_minibasket_click INTEGER,
        checked_delivery_detail INTEGER,
        checked_returns_detail INTEGER,
        sign_in INTEGER,
        saw_checkout INTEGER,
        saw_sizecharts INTEGER,
        saw_delivery INTEGER,
        saw_account_upgrade INTEGER,
        saw_homepage INTEGER,
        device_mobile INTEGER,
        device_computer INTEGER,
        device_tablet INTEGER,
        returning_user INTEGER,
        loc_uk INTEGER,
        ordered INTEGER,
        created_at TIMESTAMP
    );
    , parameters: None
[2024-06-07T14:01:59.233+0000] {logging_mixin.py:188} INFO - Create table successful
[2024-06-07T14:01:59.240+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-07T14:01:59.243+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-07T14:01:59.263+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=s3_postgres, task_id=create_postgres_table, run_id=scheduled__2024-06-05T00:00:00+00:00, execution_date=20240605T000000, start_date=20240607T140156, end_date=20240607T140159
[2024-06-07T14:01:59.377+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-07T14:01:59.549+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-07T14:01:59.555+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
